knot.x/knotx/conf/routes/handlers/scenario0.conf

```diff
@@ -0,0 +1,88 @@
+debugMode = false
+
+tasks {
+  payment-check {
+    action = user
+    onTransitions {
+      _success {
+        actions = [
+          {
+            action = creditCard
+          }
+          {
+            action = paypal
+          }
+          {
+            action = payU
+          }
+        ]
+        onTransitions {
+          _success {
+            action = payments
+            onTransitions {
+              _success {
+                action = template-engine-handlebars
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+actions {
+  payments {
+    factory = payments
+  }
+  user {
+    factory = http
+    config {
+      endpointOptions {
+        path = /scenario0/user
+        port = 3080
+        allowedRequestHeaders = ["Content-Type"]
+      }
+    }
+  }
+  creditCard {
+    factory = http
+    config {
+      endpointOptions {
+        path = /scenario0/creditcard/allowed
```

Comment:
I think it would be good to have here some placeholder as it will be common use, do you agree?